src/pages/HomePage/ListOfHouses/ListOfHouses.jsx

```diff
@@ -1,22 +1,57 @@
+/* eslint-disable import/no-absolute-path */
+/* eslint-disable import/no-unresolved */
+/* eslint-disable react/prop-types */
 /* eslint-disable react/jsx-props-no-spreading */
 import { Favorite, FavoriteBorder } from '@material-ui/icons';
 import DoneIcon from '@mui/icons-material/Done';
 import { Autocomplete, Box, Button, Checkbox, TextField } from '@mui/material';
 import Chip from '@mui/material/Chip';
-import React from 'react';
+import { useEffect, useRef, useState } from 'react';
+
+import noPhoto from '/src/assets/images/nophoto.png';
 
 import styles from './ListOfHouses.module.scss';
 
-const options = ['Option 1', 'Option 2'];
+const options = ['Payment (Low to High)', 'Payment (High to Low)', 'A-Z', 'Z-A'];
 const label = { inputProps: { 'aria-label': 'Checkbox demo' } };
 
-function ListOfHouses() {
-  const [value, setValue] = React.useState(null);
+function ListOfHouses({ houses }) {
+  const [sortType, setSortType] = useState(null);
+  const [sortedHouses, setSortedHouses] = useState([]);
+  const houseList = useRef([]);
 
   const handleDelete = () => {
-    setValue(null);
+    setSortType(null);
   };
 
+  useEffect(() => {
+    houseList.current = houses;
+    setSortedHouses(houseList.current);
+  }, [houses]);
+
+  useEffect(() => {
+    const itemArray = houseList.current.map((item) => item);
```

Comment:
This could be changed to `houses.map((item) => item);` if you remove `houseList`.